docs/dashboard/catalog/products.md

```diff
@@ -211,7 +211,9 @@ Categorize your new product using the drop-down menus to assign a relevant Produ
 
 Once you have set up all your product details, you need to assign a previously created [channel](dashboard/configuration/channels.md). Click the _MANAGE_ button in the top right corner of the availability card, and then assign a channel. You can assign as many channels as you need for each product.
 
-Each channel gives you fine control over the availability of individual items. You can set a product's publication date on a per-channel basis.
+Each channel gives you fine control over the availability of individual items. You can choose all or specific variants that will be assigned to a channel. Then you can set a product's publication date on a per-channel basis.
+
+![](../screenshots/catalog-product-variant-availability.png)
```

Comment:
The screenshot doesn't look too good for two reasons:
- scrollbars
- channel name is "NightCity" - is this a good example for the docs? I would say something neutral like "Default Channel" or "Channel USD"

What browser do you use to take screenshots? I'm using Chrome on Mac and I don't see these scrollbars:
![image](https://user-images.githubusercontent.com/5421321/118796317-ca652500-b89b-11eb-8462-e081bd437090.png)

Reply:
I'm using Chrome on Windows. Yes, we've used "NightCity" in other screenshots and assets as an easter egg--I can change it.

Reply:
@maarcingebala It's a bug. There is already a ticket for it, but as far as I know, it has a low priority.

Reply:
While the bug is waiting for a fix, can't we use a screenshot made on Mac?